lib/tasks/one_off/merge_user_into_another.rake

```diff
@@ -0,0 +1,30 @@
+namespace :one_off do
```

Comment:
It is not technically a one off as this script will be runned few times I guess?

Reply:
👍🏽 good point - I'll move this out of one_off